jwst/assign_wcs/nirspec.py

```diff
@@ -72,7 +72,7 @@ def imaging(input_model, reference_files):
     dircos2unitless = DirCos2Unitless(name='directional_cosines2unitless')
 
     col_model = CollimatorModel(reference_files['collimator'])
-    col = col.model
+    col = col_model.model
```

Comment:
Good catch @jdavies-st ! May be this will fix other problems I see with CV3 data.